src/assets/style/loadingView.css

```diff
@@ -0,0 +1,68 @@
+/*#a3e3e8*/
+/*#4a858b*/
+/*#274f55*/
+/*#0d1b1e*/
+
+.loadingView {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    background-color: #0d1b1e;
+    color: #dddddd;
+    z-index: 1500;
+}
+
+.loadingViewMedia {
+    width: 500px;
+    overflow: hidden;
+}
+
+.loadingViewMedia-figure {
+    float: left;
+    margin-right: 50px;
+}
+
+.loadingViewMedia-content {
+    margin-top: 20px;
+    overflow: hidden;
+}
+
+.loadingViewMedia-content > * + * {
+    margin-top: 5px;
+}
+
+.loadingViewMedia-bd {
+    color: #a3e3e8;
+    font-size: 80%;
+}
+
+.loadingViewMedia-ft {
+    color: #888888;
+    font-size: 70%;
+}
```

Comment:
I feel like this div could maybe use some margin, it feels a bit tight to the lines above, no?